external/error_report/Kconfig

```diff
@@ -0,0 +1,67 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config ERROR_REPORT
+	bool "Error Report Module"
+	default n
+	---help---
+		enable Error Reporting
+
+if ERROR_REPORT
+
+config ERROR_REPORT_NENTRIES
+	int "Number of Error Records"
+	default 6
+	---help---
+		Maximum number of error records to store in memory
+
+config DEBUG_ERR_REPORT_ERROR
+	bool "Debug Error Reporting for errors"
+	depends on DEBUG
```

Comment:
You need to change dependancy configs for error, warn and info to DEBUG_ERROR, DEBUG_WARN and DEBUG_VERBOSE.
Please refer network debug configs.
```
config DEBUG_NET_ERROR
	bool "Network Error Output"
	default n
	depends on DEBUG_ERROR
	---help---
		Enable network error debug SYSLOG output.

config DEBUG_NET_WARN
	bool "Network Warning Output"
	default n
	depends on DEBUG_WARN
	---help---
		Enable network warning debug SYSLOG output.

config DEBUG_NET_INFO
	bool "Network Informational Debug Output"
	default n
	depends on DEBUG_VERBOSE
	---help---
		Enable network informational debug SYSLOG output.
```